### PR TITLE
docs: Small changes to API Gateway docs

### PR DESCRIPTION
**Pull request by david-yu**
Provide link to Helm chart apigateway stanza and clarify a few requirements.

### CODE REVIEW

File: website/content/docs/api-gateway.mdx [hunk 1]
```diff
@@ -38,7 +38,7 @@ Your datacenter must meet the following requirements prior to configuring the Co
 
    </CodeBlockConfig>
 
-1. Create a values file for your Consul server agents that contains the following parameters:
+1. Create a `values.yaml` file for your Consul API Gateway deployment that contains the following configuration through the Consul Helm chart. See [Helm Chart Configuration - apigateway](https://www.consul.io/docs/k8s/helm#apigateway) for more available options on how to configure your Consul API Gateway deployment via the Helm chart.
```
Comment (karl-cardenas-coding) [inline website/content/docs/api-gateway.mdx:41]:
```suggestion
1. Create a `values.yaml` file for your Consul API Gateway deployment. Copy the content below into the `values.yaml` file. The `values.yaml` will be used by the Consul Helm chart. See [Helm Chart Configuration - apigateway](https://www.consul.io/docs/k8s/helm#apigateway) for more available options on how to configure your Consul API Gateway deployment via the Helm chart. 
```
The first sentence is a run-on sentence, so I broke it up into three sentences that makes it easier for the reader to follow along.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/570917.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit c84fe812c5e67c466f95f5ffb82783a48bc990cb onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit c84fe812c5e67c466f95f5ffb82783a48bc990cb onto `release/1.11.x` succeeded!

**Pull request merged**